### ISSUE TITLE
Google Docs integration

### ISSUE THREAD

**Issue by kelvinst** [techgaun/ex_google#2]
Are you open to PRs? 

Because I need to implement something to import a google spreadsheets and the libs I already found are very opinionated on the way to connect with google! One forces you to make the spreadsheet public, but the spreadsheet is not mine, and the other one forces you to create a service account and share the spreasheet through it, but again, the spreadsheet is not mine! 

This lib is the best for Google API integrations for what I've searched and I will contribute with some Google Docs integration if you allow me!

**Comment by kelvinst** [comment]
PS.: I'll need some token generation for private spreadsheets access too.

**Comment by kelvinst** [comment]
> PS.: I'll need some token generation for private spreadsheets access too.

Forget it! Ueberauth is good enough for OAuth2 token generation. I'll implement the Google SpreadSheet API client and get back with a PR soon!

**Comment by techgaun** [comment]
@kelvinst yeah I am definitely open to PRs. I would be happy to take your PR in for supporting spreadsheet stuff. thanks!
